src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -37,13 +37,8 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
     }
 
     const reconnection = existingConnection || await initializeConnection(options, logger);
-    if (reconnection.state !== signalR.HubConnectionState.Connected) {
```

Comment:
https://github.com/aspnet/AspNetCore/pull/13015#discussion_r314131216

Reply:
So do we need to do something here to throw? Who's actually doing the throwing in this case?

Reply:
SignalR's throwing, I considered throwing a custom exception but I felt the need to back this out after the throwing from SignalR seemed to absolutely be intentional.

Reply:
You sound extra surprised though. How do you feel about me throwing a custom exception?